src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.test.js

```diff
@@ -0,0 +1,512 @@
+      };
+
+      expect(ExpandAllButton).toHaveBeenNthCalledWith(expandAllButtonCallOrder.render, expect.objectContaining(expectedProps), {});
+    });
+
+    it('"ExpandAllButton" should be updated after clicking on it', () => {
+      const expectedProps = {
+        accordionStatus: {
+          generalFixedDueDate: !defaultState.sections.generalFixedDueDate,
+          schedule: !defaultState.sections.schedule,
+        },
+      };
+
+      fireEvent.click(container.getByTestId('expand-all-button'));
+
+      expect(ExpandAllButton).toHaveBeenNthCalledWith(expandAllButtonCallOrder.update, expect.objectContaining(expectedProps), {});
+    });
+
+    it('Footer should contain "Cancel" button with appropriate props', () => {
+      const expectedProps = {
+        id: 'clickable-cancel-fixedDueDateSchedule',
+        onClick: defaultProps.onCancel,
+      };
+
+      expect(Button).toHaveBeenNthCalledWith(buttonCallOrder.cancel, expect.objectContaining(expectedProps), {});
+      expect(container.getByText(messageIds.cancel)).toBeInTheDocument();
+    });
+
+    it('Footer should contain "Save and close" button with appropriate props', () => {
+      const expectedProps = {
+        id: 'clickable-save-fixedDueDateSchedule',
+        disabled: defaultProps.pristine,
+      };
+
+      expect(Button).toHaveBeenNthCalledWith(buttonCallOrder.save, expect.objectContaining(expectedProps), {});
+      expect(container.getByText(messageIds.saveAndClose)).toBeInTheDocument();
+    });
+
+    it('"AccordionSet" should be called with correct props', () => {
+      const expectedProps = {
+        accordionStatus: defaultState.sections,
+        onToggle: expect.any(Function),
+      };
+
+      expect(AccordionSet).toHaveBeenNthCalledWith(accordionSetCallOrder.render, expect.objectContaining(expectedProps), {});
+    });
+
+    it('"AccordionSet" should be called with updated props after clicking on toggle button', () => {
+      const expectedProps = {
+        accordionStatus: {
+          generalFixedDueDate: !defaultState.sections.generalFixedDueDate,
+          schedule: defaultState.sections.schedule,
+        },
+        onToggle: expect.any(Function),
+      };
+
+      fireEvent.click(container.getByTestId('accordionSet'));
+
+      expect(AccordionSet).toHaveBeenNthCalledWith(accordionSetCallOrder.update, expect.objectContaining(expectedProps), {});
+    });
+  });
+
+  describe('Render of edit form', () => {
+    let container;
+    const buttonCallOrder = {
+      delete: 2,
+      save: 3,
+      updatedDelete: 5,
+    };
+
+    const initialValues = {
+      id: 'test-id',
+      name: 'test-name',
+      metadata: {
+        createdDate: 'test-date',
+      },
+    };
+
+    const props = {
+      ...defaultProps,
+      pristine: false,
+      initialValues,
+    };
+
+    beforeEach(() => {
+      container = render(
+        <FixedDueDateScheduleForm {...props} />
+      );
+    });
+
+    afterEach(() => {
+      Button.mockClear();
```

Comment:
Looks like you have global `jest.clearAllMocks();` so we do not need to clear button one more time. Please check it.